string PANEL_POWER_COVER
string PANEL_POWER_SWITCH
string PANEL_POWER_LED
string A01_ROCKER_SWITCH
string A01_ROCKER_LED_TOP
string A01_ROCKER_LED_BOTTOM
string A02_LED_NUM_PAD_A1
string A02_LED_NUM_PAD_A2
string A02_LED_NUM_PAD_A3
string A02_LED_NUM_PAD_B1
string A02_LED_NUM_PAD_B2
string A02_LED_NUM_PAD_B3
string A02_LED_NUM_PAD_C1
string A02_LED_NUM_PAD_C2
string A02_LED_NUM_PAD_C3
string A02_NUM_PAD_A1
string A02_NUM_PAD_A2
string A02_NUM_PAD_A3
string A02_NUM_PAD_B1
string A02_NUM_PAD_B2
string A02_NUM_PAD_B3
string A02_NUM_PAD_C1
string A02_NUM_PAD_C2
string A02_NUM_PAD_C3
string A03_TOGGLE
string A03_LED
string A04_TOGGLE
string A04_LED_TOP
string A04_LED_BOTTOM
string A05_TOGGLE
string A05_LED
